x-pack/plugins/infra/public/components/alerting/inventory/expression_lazy.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Expressions } from './expression';
+
+// eslint-disable-next-line import/no-default-export
```

Comment:
Is there a reason to not do the default export in `./expression`? I understand this is necessary for `React.lazy()` but the extra step seems... extra. 

I wonder if something like `React.lazy(() => import('./expression').then(({ Expressions }) => Expressions))` would work, (or if it would read nicely)

Reply:
I could `export default` from the file directly, but felt the desire to isolate the rule violation. I also hope that it'll support the upcoming tree-shaking algorithm better.

The latter idea wouldn't work according to the webpack docs:

> The reason we need `default` is that since webpack 4, when importing a CommonJS module, the import will no longer resolve to the value of `module.exports`, it will instead create an artificial namespace object for the CommonJS module.

Reply:
I moved the default export into the original files.